deps/verifier/Cargo.toml

```diff
@@ -65,9 +70,10 @@ scroll = { version = "0.13.0", default-features = false, features = [
 serde.workspace = true
 serde_json.workspace = true
 serde_with = { workspace = true, optional = true }
-sev = { version = "6.3.1", default-features = false, features = [
+sev = { git = "https://github.com/virtee/sev", rev = "d487809188c3c92e397bc92e2cb2a85eb982c980", default-features = false, features = [
```

Comment:
There's no new release with the fix yet?

Reply:
nope